### PR TITLE
Remove dynamic domains from survey

### PR DESCRIPTION
**Pull request by xmo-odoo**
Task 2115472

### CODE REVIEW

**Comment by xmo-odoo** [comment]
ping @tde-banana-odoo @awa-odoo transformation seems correct to me but can you check? My understanding is anyone can be invited to take the survey if signup is possible *or* the survey doesn't require a login, otherwise only partners with a user_id can be invited to take the survey.

File: addons/survey/wizard/survey_invite.py [hunk 1]
```diff
@@ -43,7 +43,12 @@ def _get_default_author(self):
         help="Author of the message.")
     # recipients
     partner_ids = fields.Many2many(
-        'res.partner', 'survey_invite_partner_ids', 'invite_id', 'partner_id', string='Recipients')
+        'res.partner', 'survey_invite_partner_ids', 'invite_id', 'partner_id', string='Recipients',
+        domain="""[
+            '|', (survey_users_can_signup or not survey_users_login_required, '=', 1),
```
Comment (tde-banana-odoo) [inline addons/survey/wizard/survey_invite.py]: '|', '|',
(survey_users_can_signup, '=', 1)
(not survey_users_login_required, '=', 1)
('user_ids', '!=', False)

Comment (xmo-odoo) [reply addons/survey/wizard/survey_invite.py]: Works for me.

**Comment by xmo-odoo** [comment]
robodoo r+

**Pull request closed without merging**